Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -3,69 +3,217 @@
 
 using UnityEngine;
 using UnityEngine.Events;
+using System;
+using UnityEngine.Serialization;
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
     /// <summary>
-    /// A game object with the "Target" script attached can be selected and may react to hover states.
+    /// A game object with the "EyeTrackingTarget" script attached reacts to being looked at independent of other available inputs.
     /// </summary>
-    public class EyeTrackingTarget : BaseEyeFocusHandler, IMixedRealityPointerHandler, IMixedRealitySpeechHandler
+    public class EyeTrackingTarget : InputSystemGlobalListener, IMixedRealityPointerHandler, IMixedRealitySpeechHandler
     {
+        [Tooltip("Select action that are specific to when the target is looked at.")]
         [SerializeField]
         private MixedRealityInputAction selectAction = MixedRealityInputAction.None;
 
+        [Tooltip("List of voice commands to trigger selecting this target only if it is looked at.")]
         [SerializeField]
-        private MixedRealityInputAction[] voice_select = null;
+        [FormerlySerializedAs("voice_select")]
+        private MixedRealityInputAction[] voiceSelect = null;
 
-        [Tooltip("If true, the game object is set to the center of the currently looked at target.")]
-        public bool cursor_snapToCenter = false;
-
-        [Tooltip("Event is triggered when the user starts to look at the target.")]
+        [Tooltip("Duration in seconds that the user needs to keep looking at the target to select it via dwell activation.")]
+        [Range(0, 10)]
         [SerializeField]
-        private UnityEvent OnLookAtStart = null;
+        private float dwellTimeInSec = 0.8f;
+
+        [Tooltip("Event is triggered when the user starts to look at the target.")]        
+        public UnityEvent OnLookAtStart = null;
```

Comment:
Can we leave these as private and add public accessors instead? We've wanted to keep things encapsulated, [per the coding guidelines](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Documentation/Contributing/CodingGuidelines.md#encapsulation).

Might also be a good time to make the serialized field lowercase with the `FormerlySerializedAs` tag.

Reply:
@keveleigh Updated.